content/10.diagnostics.md

```diff
@@ -357,12 +358,12 @@ Several technologies have been used to develop serological tests for COVID-19, i
 
 The application of ELISA to serological testing is complementary to its use in molecular diagnostics (see above).
 Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.
-On March 18, 2020, the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the RBD of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713; @doi:10.1038/s41591-020-0913-5].
-A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD have reported a specificity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) [@doi:10.3389/fmicb.2020.618097].
+In March 2020, the Krammer lab proposed an ELISA test that detects IgG and IgM that react against the receptor-binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1038/s41591-020-0913-5].
+A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD reported a selectivity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) [@doi:10.3389/fmicb.2020.618097]. <!-- I suggest not using selectivity here or elsewhere, we should standardize terminology if that is meant as a synonym of specificity -->
```

Comment:
Agreed, that's why I changed the few instances to specificity in the other PR. This seems to inadvertently revert it?

Reply:
Fixed in c1298eea0dcc728014917c3ca9428558eba98072